app/models/product_import/product_importer.rb

```diff
@@ -214,7 +214,9 @@ def open_spreadsheet
     end
 
     def accepted_mimetype
-      File.extname(@file.path).in?('.csv', '.xls', '.xlsx', '.ods') ? @file.path.split('.').last.to_sym : false
+      return false unless ['.csv', '.xls', '.xlsx', '.ods'].include? File.extname(@file.path)
```

Comment:
This is a great change which should go to master. There is actually a conflict already because we removed the options to `.csv` only.